pkg/Library/Graphs/Demos/AIClock.js

```diff
@@ -0,0 +1,364 @@
+export const graph = {
```

Comment:
there is also `pkg/GraphsDemos/` folder... i think there should only be one?

Reply:
Yes, but I wasn't sure if you had dependencies on the old one.

Reply:
sorry, i missed this comment. no, i don't think there are dependencies on the Graphs/Demos files in general, these are just a precaution against losing them (maybe they shouldn't be in this repo at all? dunno)...
anyway, i'll remove the duplicates in a followup PR.